### PR TITLE
Fixes #485 : Make changes in the Skills Card on Skills home page

### LINKED ISSUE #485: Make changes in the Skills Card on SUSI Skills home page
**Actual Behaviour**
It shows hands-up/down rating

**Expected Behaviour**
Change it to the 5-star rating system.

**Steps to reproduce it**
Go to http://susi.skills.ai

**Screenshots of the issue**
* Actual
screenshot from 2018-05-24 09-40-48

* Expected
screenshot from 2018-05-24 09-41-02

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by akshatnitd**
Fixes #485 

Changes: 
Make changes in the Skills Card on Skills home page & make it similar to that of Alexa Skills Card

Surge Deployment Link: https://pr-496-fossasia-susi-sill-cms.surge.sh

Screenshots for the change:
screenshot from 2018-05-25 05-32-17

### CODE REVIEW

**Comment by DNS-404** [comment]
@akshatnitd Look -
screen shot 2018-05-25 at 8 58 35 am

File: src/components/BrowseSkill/BrowseSkill.js [hunk 1]
```diff
@@ -295,13 +294,20 @@ export default class BrowseSkill extends React.Component {
...
                                 <div style={styles.rating}>
-                                    <span style={styles.positive}>
-                                        <Like style={styles.like} />
-                                        {positive_rating}
-                                    </span>
-                                    <span style={styles.negative}>
-                                        <Dislike style={styles.dislike} />
-                                        {negative_rating}
+                                    <Ratings
+                                        rating={average_rating || 2.4}
```
Comment (anshumanv) [inline src/components/BrowseSkill/BrowseSkill.js]: Let's use 0 as a default instead. :thinking:

Comment (akshatnitd) [reply src/components/BrowseSkill/BrowseSkill.js]: @anshumanv Yes, I think there was some error while committing. Making the change right away!

**Comment by akshatnitd** [comment]
@DNS-404 @Akshat-Jain Yes, the rating will be displayed, once the server side is done.

File: src/components/BrowseSkill/BrowseSkill.js (hunk 1 above)
Comment (anshumanv) [reply src/components/BrowseSkill/BrowseSkill.js]: Cool thanks! :smile:

**Comment by akshatnitd** [comment]
@mariobehling Shall I also remove the skill description ?

**Comment by mariobehling** [comment]
Ok, let's get some experience with this. Thank you.

**Pull request merged**